src/util/mod.rs

```diff
@@ -89,7 +90,9 @@ pub enum Error {
     /// Error propagated from subsystem
     Detail(String, Box<Error>),
     /// Unsupported witness version
-    UnsupportedWitnessVersion(u8)
+    UnsupportedWitnessVersion(u8),
+    /// PSBT-related error
+    PSBT(psbt::Error),
```

Comment:
Shouldn't this variant be named `Psbt`? All types with acronyms in the codebase only have the first letter uppercased e.g. `Sha256dHash`, `Hash160` ([C-CASE][1]).

[1]: https://rust-lang-nursery.github.io/api-guidelines/naming.html#c-case

Reply:
See 7a6df92 and 5168c7c, will rebase

Reply:
Done